incompatibility/incompatibility-report/incompatibility-report.pb.go

```diff
@@ -0,0 +1,559 @@
+// Copyright 2017 Google LLC. All Rights Reserved.
```

Comment:
The general approach for code generated from protos is not to commit the files, but generate as part of the build phase.

Reply:
removed and updated gitignore